umbra-js/src/classes/Umbra.ts

```diff
@@ -27,7 +27,7 @@ import {
 import { KeyPair } from './KeyPair';
 import { RandomNumber } from './RandomNumber';
 import {
-  blockedStealthAddresses,
```

Comment:
*random placement for comment*

Receiving is not working for me on local and deploy preview on Brave & arbitrum 
![image](https://github.com/ScopeLift/umbra-protocol/assets/61768337/56b74fbe-199a-4a0e-93e2-b7960e3f07a8)

Reply:
Get this additional warning at the start on optimism
![image](https://github.com/ScopeLift/umbra-protocol/assets/61768337/66631de7-d751-4aa9-979e-a058046c3b9d)

Reply:
Ok should be fixed in e459dcf, but needs more testing. Part of the testing should including manually overriding an announcement's receiver and/or from address and make sure it gets filtered out of the receive table

Reply:
Looks like the node is still being overwhelmed, at least at the moment on the deploy preivew 
<img width="1347" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/442969d9-75bc-4ef5-a9aa-2eaeec2bb2ff">

Reply:
Hm I tested locally with alchemy and didn't see that. I'll try mainnet later to mimic what you have. That still seems odd though. What's the batch size that announcements are processed as? I believe we now make one call per `filterSupportedAddresses` call